docs/GDI-SOP_organisational-roles-and-responsibilities.md

```diff
@@ -0,0 +1,56 @@
+- **GDI Node Service Operators**: Personnel involved in testing, implementing, and operating GDI services according to the SOPs.
+- **GDI Governance Bodies**: Committees like the Operations Committee (OC) and Security and Data Protection Committee (SDPC) responsible for overseeing the SOP lifecycle.
+
+## 6. Roles and Responsibilities Definitions
+
+| Role | Relevant at | Primary Responsibilities | Skills and Qualifications |
+| -- | -- | -- | -- |
+| **Author**   | European and Node level | - **Fill in content of SOP Templates** based on expertise and input from stakeholders across GDI nodes. <br> - Ensure **compliance with data policies** (e.g., GDPR). <br> - Take the role of "SOP Trainer" for their authored SOP Templates if required. | - Expertise in the relevant area. <br> - Strong communication and collaboration skills. <br> - Knowledge of GDPR and data handling requirements. |
+| **Reviewer** | European and Node level | - **Review SOP Templates** for accuracy and compliance. <br> - Suggest improvements and ensure alignment with GDI objectives. <br> - Facilitate inter-node communication. | - Broad understanding of GDI's objectives. <br> - Detail-oriented with a critical eye for issues. <br> - Experience in collaborative reviews and multi-disciplinary teams. |
+| **Approver** | European and Node level | - **Confirm SOP Templates readiness** post-review. <br> - **Ensure compliance** with **GDI standards and GDPR**. <br> - Document and store SOP Templates in designated platforms (i.e., GitHub). <br> - Oversee SOP lifecycle and ensure periodic reviews. <br> - Transfer approved SOPs to GitHub for version control.  | - Leadership and decision-making capabilities. <br> - Deep understanding of GDI strategic goals. <br> - Proficiency in managing project documentation platforms. <br> - Experience in document lifecycle management. |
+| **Authorizer** | European level | - **Grant (or veto) official permission for SOPs to be released**. | - High-level management skills. <br> - Understanding of GDI scope and operations.  |
+| **Main repository Maintainer (OC/SDPC)** | European level | - **Oversee SOP creation and review process** for the GDI GitHub repository. <br> - **Coordinate with Authors, Reviewers, Approvers and Authorizers** to ensure SOP templates meet GDI standards. <br> - Manage version control, ensure updates are made consistently. <br> - **Conduct periodic audits** of the SOP repository to maintain quality and compliance. | - Strong project management and oversight skills. <br> - Proficiency in GitHub repository management, including version control and collaborative document handling. <br> - Comprehensive understanding of GDI operations and objectives. <br> - Ability to work effectively with multidisciplinary teams and manage stakeholder expectations. |
```

Comment:
Isn't there a certain overlap with the responsibilities of the Approver (version control & overseeing SOP life-cycle)??

Reply:
Addressed above